packit/api.py

```diff
@@ -746,6 +748,15 @@ def push_updates(self, update_alias: Optional[str] = None):
                 logger.debug(f"{update['alias']} is not ready to be pushed to stable")
 
     def init_kerberos_ticket(self) -> None:
+        """
+        Initialize the kerberos ticket if we have fas_user and keytab_path configured.
+        """
+        if self._kerberos_initialized:
+            return
+        self._init_kerberos_ticket()
+        self._kerberos_initialized = True
+
+    def _init_kerberos_ticket(self) -> None:
```

Comment:
I find these almost similar method names very confusing when reading the code, especially with the doc-strings being identical.

Why not just update the old `init_kerberos_ticket()` to have the check for and the setting of `_kerberos_initialized`?

Reply:
I'll update the naming.

Reason? Easier testing. (We can't check the real command execution since it's not run when the authentication is not configured.)